### DISCUSSION TITLE
RSS Atom module is caching downloaded results without any means of reducing the time or deactivating caching

### DISCUSSION THREAD

**Discussion by ianpmurphy** [grafana/grafana#64892] [Dashboards, Panels & Visualizations]
**Feature request**

**The problem**
The RSS/Atom module works perfectly except for a small problem. The RSS flow is downloaded and cached for a very long time. It seems to be over 24 hours. In an corporate environment with an internal rss feed this will mean that news updates don't appear after publication. From the users perspective, if they publish a news article to a feed and then go to the dashboard to view the change, it should appear. Currently changes which are published today will not appear until tomorrow.
At the same time, the Timeseries module displays accurate data every 5 minutes.

I've tried everything I can control to change this behaviour but it seems to be a functionality of the module itself.

When grafana is used on a lan there really isn't any advantage to caching rss results as the module could pull down on every dashboard refresh.

**The Request**
I'd like to request that this be made either a cache time parameter on the module or made into a checkbox to simply disable caching completely.

**Comment by gelicia** [comment]
The RSS/Atom datasource is not mainatained by us. I think this question would be better asked in the repo for it https://github.com/VolkovLabs/volkovlabs-rss-datasource

**Reply by ianpmurphy** [reply]
Isn't the caching handled by Grafana?

**Reply by gelicia** [reply]
I am not super familiar with the plugin, but generally plugins handle their own data, including caching. Grafana just gives a platform for displaying the panel.

**Reply by ianpmurphy** [reply]
ah, I thought everything flowed through grafana and plugins just had to know how to be passed data. Obviously not.
thanks in any case.

**Reply by gelicia** [reply]
No problem - and also I could be wrong. I think reaching out to that repo is the best next step, but if it does end up being a Grafana bug that they help you define, feel free to log it in our [issues](https://github.com/grafana/grafana/issues).
